src/__namespace.js

```diff
@@ -1,4 +1,5 @@
 /**
  * @ignore
  */
-wikibase.datamodel = wikibase.datamodel || {};
+window.wikibase = window.wikibase || {};
```

Comment:
It fails on NodeJS because `wikibase` variable is not defined and it tries to access it.
But having more full explanation would be nice.